Fix python executable name (passed via robotpkg)
Fix installation dir

--- urdf_parser_py/CMakeLists.txt~	2014-10-14 11:12:50.492913082 +0200
+++ urdf_parser_py/CMakeLists.txt	2014-10-14 11:13:39.356736044 +0200
@@ -1,7 +1,6 @@
-find_program(PYTHON "python")
 
-if (PYTHON)
+if (PYTHON_EXECUTABLE)
   set(SETUP_PY "${CMAKE_CURRENT_SOURCE_DIR}/setup.py")
-  install(CODE "execute_process(COMMAND \"${PYTHON}\" \"${SETUP_PY}\" build --build-base \"${CMAKE_CURRENT_BINARY_DIR}/pybuild\" install --install-layout deb --prefix \"${CMAKE_INSTALL_PREFIX}\"
+  install(CODE "execute_process(COMMAND \"${PYTHON_EXECUTABLE}\" \"${SETUP_PY}\" build --build-base \"${CMAKE_CURRENT_BINARY_DIR}/pybuild\" install --install-lib=${CMAKE_INSTALL_PREFIX}/${PYTHON_SITELIB} --prefix \"${CMAKE_INSTALL_PREFIX}\"
                 WORKING_DIRECTORY \"${CMAKE_CURRENT_SOURCE_DIR}\")")
 endif()
